lib/vue-simple-suggest.vue

```diff
@@ -506,13 +507,20 @@ export default {
     onInput (inputEvent) {
       const value = !inputEvent.target ? inputEvent : inputEvent.target.value
 
+      this.$emit('input', value)
+      this.updateTextOutside(value)
+    },
+    updateTextOutside(value) {
       if (this.text === value) { return }
 
       this.text = value
-      this.$emit('input', this.text)
-
```

Comment:
Moving the location of 'input' emit could have unintended side-effects here. Need to double-check this.

Reply:
My intention was to simplify code

I see 2 things here:
- $emit fires before checking for value change
Looks like the only way to make it - copy-paste same value to the input, I think it's OK and nothing wrong to notify parent component that user made this action

- $emit fires before `text` update
I missed this point, and I agree that this case looks pretty bad, because emitting input will cause `value` prop update, and we must be sure that `text` should be updated before this. I intend to update it

Reply:
I have moved `$emit('input')` lower than `text` update